src/components/tile/tile.stories.mdx

```diff
@@ -202,38 +202,46 @@ Please note that it will not have any effect if the Tile orientation is set to `
 
 <Canvas>
   <Story name="custom Widths">
+    <div>
```

Comment:
Shouldn't we move the stories themselves into a `tile.stories.tsx` file?

Reply:
@mkrds did already asked for the same thing, please check his second comment:
https://github.com/Sage/carbon/pull/5738#discussion_r1068181986